### ISSUE TITLE
[BUG] Reattaching heads don't work and a living brain won't be detected

### ISSUE THREAD

**Issue by AGasCanister** [DS-13-Dev-Team/DS13#400]
#### Description of issue
When a guy gets decapped and trying to put his head back, the game won't detect the living brain and keeping the guy dead.
If you reattach a head back and scan him with the body scanner, the brain won't be called "Brain" it will be called "X's brain" 

#### Difference between expected and actual behavior
In normal servers that have brainmed, you can reattach the guy's head back and get him back alive

#### Steps to reproduce 
1- Get a human
2- Decap that human
3- Reattach his head
4- He is still brain dead even on the body scanner his brain appears healthy

#### Specific information for locating

Brain, Human

#### Length of time in which bug has been known to occur

Basically in every round where there are humans

#### Client version, Server revision & Game ID

Client Version: 513
Server Revision: 7e173db25ac49eaed940c07d4e3b2ad8318f4f61 - dev - 2020-05-26
Game ID: b7l-dhtV
Current map: USG Ishimura

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of the issue being looked at sooner.

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/Baystation12/Baystation12/issues)

**Comment by Snypehunter007** [comment]
TBF, I don't think re-attaching is a viable thing in DS.

**Comment by spicyfly** [comment]
heads maybe could be a viable source of necro something in the future! but hooman must die

**Comment by NanakoAC** [comment]
Is this really a bug? being decapitated is not meant to be survivable

**Comment by Snypehunter007** [comment]
Maybe more so that the scanner says his brain is healthy?

IDK.
